assets/js/data/cart/actions.ts

```diff
@@ -23,8 +23,10 @@ import { ACTION_TYPES as types } from './action-types';
 import { apiFetchWithHeaders } from '../shared-controls';
```

Comment:
Each function here will now work in the way way; returning a promise with either the successful response, or rejecting with the error message so that consumers can handle the error objects.

Reply:
@senadir There is some still in `receiveCart`, but yeah for errors its back inside hooks.